src/javaharness/java/arcs/android/demo/service/ArcsAutofillService.java

```diff
@@ -27,7 +29,8 @@
 public class ArcsAutofillService extends AutofillService {
 
   @Inject Provider<RemotePec> remotePecProvider;
-  @Inject PortableJsonParser jsonParser;
+  @Inject UiBroker uiBroker;
+  @Inject @Singleton AutofillRenderer autofillRenderer;
```

Comment:
Might be better to mark the AutofillRenderer class itself with `@Singleton`, so that you don't have to do this here.

Reply:
done in #3611